src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -43,6 +50,9 @@ export function secureCellSealWithSymmetricKeyEncrypt64(symmetricKey64, plaintex
     if (symmetricKey64 === "" || symmetricKey64 === undefined || symmetricKey64 === null) {
         throw new Error("Parameter symmetricKey64 can not be empty");
     }
+    if (!isBase64(symmetricKey64)) {
+        throw new Error("Parameter symmetricKey64 is not base64 encoded");
+    }
     const symmetricKey = Array.from(Buffer.from(symmetricKey64, 'base64'));
```

Comment:
If you're wondering why this check is necessary, `Buffer.from('💩💩💩', 'base64')` – in typical Node.js & JS fashion – simply skips all characters that it can't recognize.

Reply:
I believe if we announced that parameters should be base64 encoded, we have to validate them. Of course, I know about it behavior of Javascipt.

Reply:
That comment was for other reviewers, who might not be aware why exactly you have to validate before calling the base64 conversion.